recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  fn: holoviews-{{version}}.tar.gz
```

Comment:
@jlstevens you are using a jinja variable here `{{ version }}` that is undefined.

You can add `{% set version = "1.5.0" %}` on top of your recipe to define it. (And you can use that variable in the `version` field too.)